TROUBLESHOOTING.md

```diff
@@ -5,7 +5,13 @@
 * [How to debug log files](#debug-logs)
 
 ### Common Issues
+* [Databricks Cluster Won't Start](#cluster-fails)
 * [I don't see lineage in Microsoft Purview](#no-lineage)
+* [Databricks Cluster Won't Start](#cluster-fails)
```

Comment:
Duplicate link on line 8, which should be kept since you need the cluster before it can pass lineage